ci-scripts/ci-runner.bash

```diff
@@ -196,6 +196,10 @@ else
     # Find modified or added user checks
     userchecks=( $(git diff origin/master...HEAD --name-only --oneline --no-merges | \
                    grep -e '^cscs-checks/.*\.py') )
+    if [[ $(hostname) =~ daint ]]; then
+        # Temporarily disable running modified checks on Daint
+        userchecks=""
```

Comment:
This should be declared as an empty array, otherwise `${userchecks[@]}` equals 1.

Reply:
Sth like `userchecks=( )` ?? Thought it was easier... S**y Bash.

Reply:
Yes, `userchecks=()` should work.